integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -9,7 +9,7 @@ const getPlatformName = capabilities => {
   switch (platform) {
     case 'MAC':
       if (browserName === 'safari') {
-        return version === 'latest' ? 'macOS 11.00' : 'macOS 10.13';
+        return version === 'latest' ? 'macOS 12' : 'macOS 10.13';
       }
       return 'macOS 10.14';
```

Comment:
can we also update this value to something else oter than 10.14 which is very old?

Reply:
I'm not sure which one to use. I think this should be related to the OS we supported.

Reply:
@ltrung Since the latest version is bumped up for one version, I bumped up this for one as well.

Reply:
Tried to upgrade the default version to `macOS 10.15` or `macOS 11`. But either of them failed to pass the GitHub Action. So have to keep the default Mac version unchanged.

Reply:
I think we want to update from macOS 10.14 to BigSur. 10.14 is pretty old :)